all-the-icons.el

```diff
@@ -390,33 +390,101 @@
+    (magit-refs-mode           all-the-icons-octicon "git-branch"          :v-adjust 0.0 :face all-the-icons-red)
     (magit-process-mode        all-the-icons-octicon "mark-github"         :v-adjust 0.0)
-    (magit-diff-mode           all-the-icons-octicon "git-compare"         :v-adjust 0.0)
-    (ediff-mode                all-the-icons-octicon "git-compare"         :v-adjust 0.0)
-    (comint-mode               all-the-icons-faicon "terminal"             :v-adjust 0.0)
-    (eww-mode                  all-the-icons-faicon "firefox"              :v-adjust -0.1)
-    (org-agenda-mode           all-the-icons-octicon "checklist"           :v-adjust 0.0)
+    (magit-diff-mode           all-the-icons-octicon "git-compare"         :v-adjust 0.0 :face all-the-icons-lblue)
+    (ediff-mode                all-the-icons-octicon "git-compare"         :v-adjust 0.0 :Face all-the-icons-red)
+    (comint-mode               all-the-icons-faicon "terminal"             :v-adjust 0.0 :face all-the-icons-lblue)
+    (eww-mode                  all-the-icons-faicon "firefox"              :v-adjust -0.1 :face all-the-icons-red)
+    (org-agenda-mode           all-the-icons-octicon "checklist"           :v-adjust 0.0 :face all-the-icons-lgreen)
     (cfw:calendar-mode         all-the-icons-octicon "calendar"            :v-adjust 0.0)
-    (ibuffer-mode              all-the-icons-faicon "files-o"              :v-adjust 0.0)
+    (ibuffer-mode              all-the-icons-faicon "files-o"              :v-adjust 0.0 :face all-the-icons-dsilver)
     (messages-buffer-mode      all-the-icons-faicon "stack-overflow"       :v-adjust -0.1)
-    (help-mode                 all-the-icons-faicon "info"                 :v-adjust -0.1)
-    (benchmark-init/tree-mode  all-the-icons-octicon "dashboard"           :v-adjust 0.0)
-    (jenkins-mode              all-the-icons-fileicon "jenkins")
-    (magit-popup-mode          all-the-icons-alltheicon "git")
-    (magit-status-mode         all-the-icons-alltheicon "git")
+    (help-mode                 all-the-icons-faicon "info"                 :v-adjust -0.1 :face all-the-icons-purple)
+    (benchmark-init/tree-mode  all-the-icons-octicon "dashboard"           :v-adjust 0.0//)
+    (jenkins-mode              all-the-icons-fileicon "jenkins"            :face all-the-icons-blue)
+    (magit-popup-mode          all-the-icons-alltheicon "git"              :face all-the-icons-red)
+    (magit-status-mode         all-the-icons-alltheicon "git"              :face all-the-icons-lred)
+    (magit-log-mode            all-the-icons-alltheicon "git"              :face all-the-icons-green)
 
     ;; Special matcher for Web Mode based on the `web-mode-content-type' of the current buffer
-    (web-mode                 all-the-icons--web-mode-icon)
+    (web-mode             all-the-icons--web-mode-icon)
+
+    (fundamental-mode                   all-the-icons-fileicon "elisp"            :height 1.0 :v-adjust -0.2 :face all-the-icons-dsilver)
```

Comment:
I feel like `fundamental-mode` should be `lisp` rather than `elisp`? Does it not use Common Lisp in fundamental buffers? I may be wrong though...